src/coreclr/src/jit/gentree.h

```diff
@@ -3108,12 +3124,193 @@ struct GenTreeLclVarCommon : public GenTreeUnOp
 #endif
 };
 
+//------------------------------------------------------------------------
+// MultiRegSpillFlags
```

Comment:
These same flags were in use by multiple nodes, but with no code sharing. I'm now using them on `GenTreeLclVar` as well.